### PR TITLE
Css html

### PR DESCRIPTION
**Pull request by Stephs15**
@guicheffer @oktupol @hcarine 
here is the PR for the slides for tomorrow!

Feel free to suggest/make changes :)

Is quite a lot of information and the idea is to have a demo in between

### CODE REVIEW

**Comment by oktupol** [comment]
I merged the slides of yesterday and renamed the directory for these ones. Will look at them in detail later today :)

File: src/lessons/05_css_html/introduction_to_css.slides.html [hunk 1]
```diff
@@ -0,0 +1,346 @@
...
+        <section>
+          <!--SLIDE 2-->
+          <section data-auto-animate data-timing="10">
+            <p>But first...</p>
+            <h2>Let us recap!</h2>
+            <ul data-auto-animate data-timing="10">
+              <li class="fragment">Selectors</li>
+              <li class="fragment">Selectors</li>
+              <li class="fragment">And Selectors...</li>
+              <img class="fragment" src="./images/css.gif" />
```
Comment (oktupol) [inline src/lessons/05_css_html/introduction_to_css.slides.html]: 😂

File: src/lessons/05_css_html/introduction_to_css.slides.html [hunk 2]
```diff
@@ -0,0 +1,346 @@
...
+            <h2>Block Boxes</h2>
+            <ul>
+              <li class="fragment">Take up the entire space of the container</li>
+              <li class="fragment">Breaks into a new line</li>
+              <li class="fragment"><b>width</b> and <b>height</b> properties will be respected</li>
+              <li class="fragment">Padding, margin and order will push others away</li>
+              <li class="fragment">&lt;h1&gt;, &lt;p&gt; are block elements</li>
+            </ul>
+          </section>
+          <h2>Inline Boxes</h2>
```
Comment (oktupol) [inline src/lessons/05_css_html/introduction_to_css.slides.html:81]: There is an opening `<section>` tag missing here

File: src/lessons/05_css_html/introduction_to_css.slides.html [hunk 3]
```diff
@@ -0,0 +1,346 @@
...
+              <li class="fragment">Useful when we want to capture large amounts of text from the user</li>
+            </ul>
+            <pre>
+              <code class="fragment formattedCode">
+                <script type="text/template">
+                  <textarea name="textarea" rows="10" cols="50">
+                    Write something here
+                  </textarea>
+              </script>
+            </code>
```
Comment (oktupol) [inline src/lessons/05_css_html/introduction_to_css.slides.html]: Apparently highlight.js isn't detecting this as HTML automatically. 
```suggestion
              <code class="fragment formattedCode html">
                <script type="text/template">
                  <textarea name="textarea" rows="10" cols="50">
                    Write something here
                  </textarea>
              </script>
            </code>
```

**Review by oktupol — COMMENTED** [review]
Found a couple things, but the rest looks fine to me 👍🏽

**Comment by hcarine** [comment]
For me all seems good 😌

**Review by oktupol — APPROVED** [review]

**Pull request merged**